controllers/component_build_controller_simple_build.go

```diff
@@ -168,19 +159,43 @@ func getPipelineRunGitInfo(component *appstudiov1alpha1.Component, pacConfig map
 	if gitSourceSha == "" {
 		gitSourceSha, err = gitClient.GetBranchSha(repoUrl, revision)
 		if err != nil {
-			return nil, fmt.Errorf("failed to get git branch SHA: %w", err)
+			log.Error(err, "failed to get git branch SHA")
```

Comment:
Why the error can be ignored?

If ignore this error, `browseRepositoryAtShaLink` will be empty eventually, how should it be treated, error or ignored as well?

If this error can be ignored, it would be good to mention this info in the message.

Reply:
> Why the error can be ignored?

There is comment above. In short they are needed only for UI.

> If ignore this error, browseRepositoryAtShaLink will be empty eventually

That's expected, again this needed only for UI.